File: Minimum Number of Moves to Seat Everyone/Minimum_number.java
class Solution {
    public int minMovesToSeat(int[] seats, int[] students) {
        int n = students.length;
        Arrays.sort(students);
        Arrays.sort(seats);
        int res = 0;
        for (int i = 0 ; i<n ; i++){
            res+=Math.abs(students[i]-seats[i]);
        }
        return res ;
        
    }
}

File: Minimum Number of Moves to Seat Everyone/Minimum_number.cpp
class Solution {
public:
    int minMovesToSeat(vector<int>& seats, vector<int>& students) {
        int n = students.size();
        sort(seats.begin(), seats.end());
        sort(students.begin(), students.end());
        int res = 0;
        for(int i = 0; i<n ; i++){
            res+=abs(students[i]-seats[i]);
        }
        return res ; 

        
    }
};